src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -233,14 +233,14 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
   const promoImage = promoImageRawBlock?.model?.locator;
 
   const showTopics = Boolean(showRelatedTopics && topics.length > 0);
-
+  const authors = bylineLinkedData?.map(data => data?.authorName).join(', ');
   return (
     <div css={styles.pageWrapper}>
       <ATIAnalytics atiData={atiData} />
       <ChartbeatAnalytics
         sectionName={pageData?.relatedContent?.section?.name}
         title={headline}
-        {...(hasByline && { authors: bylineLinkedData.authorName })}
+        {...(hasByline && { authors })}
```

Comment:
You might not need the `hasByline` check here because if `bylineLinkedData` is `null`/`undefined`, then authors will also be `undefined`